plugins/modules/elastic_expedient_security_rules.py

```diff
@@ -0,0 +1,156 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+DOCUMENTATION='''
+
+module: elastic_expedient_security_rules
+
+author: Ian Scott
+
+short_description: Update Security rules to the Expedient Standard
+
+description: 
+  - Update Security rules to the Expedient Standard
+
+requirements:
+  - python3
+
+options:
+      host: ECE Host or Deployment Host
+      port: ECE Port or Deployment Port
+      username: ECE Username or Deployment Username
+      password: ECE Password or Deployment Password
+      deployment_info: (when using ECE host:port and credentials)
+        deployment_id: ECE Deployment ID
+        deployment_name: ECE Deployment Name
+        resource_type: kibana
+        ref_id: REF ID for kibana cluster, most likely main-kibana
+        version: Deployment Kibana Version
+
+'''
+from ansible.module_utils.basic import _ANSIBLE_ARGS, AnsibleModule
+
+try:
+  from ansible_collections.expedient.elastic.plugins.module_utils.kibana import Kibana
+except:
+  import sys
+  import os
+  util_path = new_path = f'{os.getcwd()}/plugins/module_utils'
+  sys.path.append(util_path)
+  from kibana import Kibana
+
+results = {}
+
+def main():
+
+    module_args=dict(   
+        host=dict(type='str',required=True),
+        port=dict(type='int', default=9243),
+        username=dict(type='str', required=True),
+        password=dict(type='str', no_log=True, required=True),   
+        verify_ssl_cert=dict(type='bool', default=True),
+        deployment_info=dict(type='dict', default=None),
+        security_rule_items=dict(type='array', default=None),
+    )
+    argument_dependencies = []
+        #('state', 'present', ('enabled', 'alert_type', 'conditions', 'actions')),
+        #('alert-type', 'metrics_threshold', ('conditions'))
+    
+    module = AnsibleModule(argument_spec=module_args, supports_check_mode=True)
+
+    results['changed'] = False
+    security_rule_items = module.params.get('security_rule_items')
+  
+    kibana = Kibana(module)
+      
+    exception_lists = kibana.get_security_exception_list()
+    target_object = None
+    
+    for exception_list in exception_lists:
+      if exception_list['list_id'] == 'endpoint_list':
+        target_object = exception_list
+        break
+    
+    if not target_object:
+      results['exception_list_status'] = 'endpoint_list was not found'
+      results['changed'] = False
+      module.exit_json(**results)
+    
+    if target_object:
+      results['exception_list_status'] = "endpoint_list found"
```

Comment:
it looks like this is just he normal status message, if you use `msg` it will print out to the console at lower levels of verbosity and as more relevant parts of the failure messages

Reply:
the output is never going to display if you're not using at least `-v` but `msg` shows at a lower level than any other key name